### PR TITLE
[malwarebytes-254] Updated code for Usage Analytics

### PR DESCRIPTION
**Pull request by rskumar-mwb**
- Updated Code for Usage Analytics.

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by ShacharKidor — APPROVED** [review]
Hi @rskumar-mwb, 
Good job!

Let's schedule a demo - I'm sending you a message on Slack.

**Comment by ShacharKidor** [comment]
@rskumar-mwb - Demo is scheduled for today at 15:30 (israel time zone).

File: Packs/Malwarebytes/Integrations/Malwarebytes/Malwarebytes.py [hunk 1]
```diff
@@ -1371,25 +1372,44 @@ def get_sa_activities_command(account_id, client_id, auth_token, hostname, path,
...
+        response = requests.request("POST", url, data=json.dumps(body), headers=headers, verify=USE_SSL)
+
+        if response.status_code == 200:
+            data = json.loads(response.text)
+            if data.get("statusCode") == 201:
+                return True
+            else:
+                return False
+        else:
+            return False
```
Comment (ShacharKidor) [inline Packs/Malwarebytes/Integrations/Malwarebytes/Malwarebytes.py:1412]: You are not using the returned booleans.
I think the previous message was better, or alternatively you should get the returned boolean in the main function and print the relevant result to a log.

**Review by ShacharKidor — CHANGES_REQUESTED** [review]

File: Packs/Malwarebytes/Integrations/Malwarebytes/Malwarebytes.py (hunk 1 above)
Comment (rskumar-mwb) [reply Packs/Malwarebytes/Integrations/Malwarebytes/Malwarebytes.py:1412]: The previous message was producing an error in the test function, that's why I changed it. found that in the morning. Also, I don't think this function needs to be logged as this is only for collecting usage analytics for us, it's not used by the customers in any way.

Comment (ShacharKidor) [reply Packs/Malwarebytes/Integrations/Malwarebytes/Malwarebytes.py:1412]: Ok :) it's your choice.

**Review by ShacharKidor — APPROVED** [review]

**Pull request merged**